mk/config.mk

```diff
@@ -582,3 +582,9 @@ CFG_SCMI_MSG_DRIVERS ?= n
 CFG_SCMI_MSG_CLOCK ?= n
 CFG_SCMI_MSG_RESET_DOMAIN ?= n
 CFG_SCMI_MSG_SMT ?= n
+
+# Path to the Python interpreter used by the build system.
+# By default the interpreter is defined by the system through /usr/bin/env.
+# Build environments that require more explicit control can configure which
+# interpreter to use through this environment variable.
+CFG_PYTHON ?= python3
```

Comment:
Wouldn't it make more sense to use `CFG_PYTHON3` instead?

Reply:
I don't know if there are situations where you can get OP-TEE to build with Python 2? I wanted to keep it flexible to the user to decide what interpreter to use.

But if there are no circumstances where Python 2 would ever be used, then yes, it probably makes more sense to name the variable `CFG_PYTHON3` .

Reply:
The current scripts are all expecting python3 as interpreter. I don't see that we'll ever add a python2 script again, we're likely to convert it python3 first.

Anyway, supplying anything but a python3 compatible interpreter in this variable is bound to break stuff so it would be good to make the expectations of this variable clear.

Reply:
+1

Reply:
Makes sense. I updated the name of the variable to better reflect these expectations.